HTML/mockup-axit/index.html

```diff
@@ -0,0 +1,445 @@
+            <picture>
+                <source media="(min-width: 1024px)" srcset="img/section-standard@3x.png">
+                <source media="(min-width: 687px)" srcset="img/section-standard@2x.png">
+                <img src="img/section-standard.png" alt="building photo">
+            </picture>
+        </div>
+    </section>
+
+    <section class="container section-offer">
+        <div class="section-offer__content">
+            <h3 class="section-heading section-heading--big">WHY THIS IS AWESOME</h3>
+            <span class="text-separator  text-separator--vertical-margin u-centered"></span>
+            <p class="section-paragraph">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+            </p>
+        </div>
+        <div class="section-offer__offer">
+            <div class="offer__item">
+                <span class="circle-icon circle-icon--big u-centered">
+                    <img src="img/icons/light-bulb.png" alt="light bulb" class="circle-icon__img">
+                </span>
+                <h4 class="offer__heading section-heading">Thoughtful Design</h4>
+                <p class="offer__paragraph section-paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing
+                    elit. Aenean euismod bibendum laoreet. Proin gravida
+                    dolor sit amet lacus accumsan et viverra.
+                </p>
+            </div>
+            <div class="offer__item">
+                <span class="circle-icon circle-icon--big u-centered">
+                    <img src="img/icons/keyboard.png" alt="light bulb" class="circle-icon__img">
+                </span>
+                <h4 class="offer__heading section-heading">Well Crafted</h4>
+                <p class="offer__paragraph section-paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing
+                    elit. Aenean euismod bibendum laoreet. Proin gravida
+                    dolor sit amet lacus accumsan et viverra.
+                </p>
+            </div>
+            <div class="offer__item">
+                <span class="circle-icon circle-icon--big u-centered">
+                    <img src="img/icons/energy.png" alt="light bulb" class="circle-icon__img">
+                </span>
+                <h4 class="offer__heading section-heading">Easy to Customize</h4>
+                <p class="offer__paragraph section-paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing
+                    elit. Aenean euismod bibendum laoreet. Proin gravida
+                    dolor sit amet lacus accumsan et viverra.
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <section class="container section-price">
+        <div class="section-price__content">
+            <h3 class="section-heading section-heading--big">PRICING OPTIONS</h3>
+            <span class="text-separator  text-separator--vertical-margin u-centered"></span>
```

Comment:
as this separator is more visual styling it would be great to have it from css with `::after`